### ISSUE TITLE
Install version 0.0.722 on Blender fails

### ISSUE THREAD

**Issue by FrankSpalteholz** [carlosedubarreto/b3d_mocap_import#20]
Hi Carlos,

when i try to install the latest version in Blender i got this error:

Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender 2.90\2.90\scripts\modules\addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "C:\Users\frank\AppData\Roaming\Blender Foundation\Blender\2.90\scripts\addons\b3d_mocap_import-main\__init__.py", line 39, in <module>
    from . load_mocap import Import_Data_easymocap, OT_TestOpenFilebrowser,Import_Data_frankmocap,Import_Data_vibe,Mediapipe_Pose_estimation, \
ImportError: cannot import name 'Audio2face_Export' from 'b3d_mocap_import-main.load_mocap' (C:\Users\frank\AppData\Roaming\Blender Foundation\Blender\2.90\scripts\addons\b3d_mocap_import-main\load_mocap.py)

before i tried to install the new version i un-installed the older one (0.0.7) via the AddOn-UI in Blender.

Maybe i was missing some other steps. 
Thanks again!

**Comment by carlosedubarreto** [comment]
There is a problem when updating, that I couldnt solve yet.

Try this, remove the previous version, closer blender, open it again, install the new version.

Probably you wont get an error.

**Comment by FrankSpalteholz** [comment]
That worked! A last side-question: you are fully on windows right? Means that you've managed to get EasyMocap running on windows too? I got a hard time get all the dependencies compiled and running on Ubuntu and i will never touch it again. But it would be good to know if it would work (in theory) to implement a "windows-only"-pipe for the future (cos its kind of tedious to switch back and forth all the time)

**Comment by carlosedubarreto** [comment]
Sure, I'm a windows only guy lol,.,,, 
My effort is usually to make it fully working on windows.

And to be honest, the installation of easymocap in windows was very easy.

The complicated part was some files tha I wasnt finding, but I noticed some of them were renamed.

I got a playlist with easymocap stuff (installation and calibration), but that is a raw recording and non scripted one.

I mean, its hard to watch 😅
I plan to record it again, but it wont be anytime soon, as now I'm working on applying the mocap_import tools on a animation project, to test it.

So here is the playlist on easymocap raw tutorials

https://youtube.com/playlist?list=PLYzax7LPWaADMqDjc9rnAOC3tsd35zZGL
